rotkehlchen/accounting/constants.py

```diff
@@ -284,6 +290,12 @@
     )}, EventCategory.CLAWBACK: {DEFAULT: EventCategoryDetails(
         label='Clawback',
         icon='lu-undo-2',
+    )}, EventCategory.MINT_NFT: {DEFAULT: EventCategoryDetails(
```

Comment:
there is also a place in the frontend may need to add them. I think it's rui/index.ts but ask them.

Reply:
there's no place in the frontend that needs updating (they retrieve all the info from the backend).

Reply:
there is but you did it